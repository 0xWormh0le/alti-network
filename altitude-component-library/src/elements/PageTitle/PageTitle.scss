.PageTitle {
  margin: 1em;
  margin-bottom: 0;
}
