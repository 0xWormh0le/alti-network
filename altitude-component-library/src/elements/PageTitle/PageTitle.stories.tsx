import React from 'react'
import { Story, Meta } from '@storybook/react/types-6-0'
import PageTitle, { PageTitleProps } from './PageTitle'

export default {
  title: 'Elements/PageTitle',
  component: PageTitle
} as Meta

const Template: Story<PageTitleProps> = (args) => <PageTitle {...args} />

export const Default = Template.bind({})
Default.args = {
  title: 'Sample Title'
}
