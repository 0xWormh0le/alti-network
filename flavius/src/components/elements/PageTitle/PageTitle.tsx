import React from 'react'
import Typography, { TypographyVariant } from 'components/elements/Typography'
import './PageTitle.scss'

export interface PageTitleProps {
  title: React.ReactNode | string
}

const PageTitle: React.FC<PageTitleProps> = ({ title }) => (
  <Typography variant={TypographyVariant.H2} component='h1' className='PageTitle' weight='normal'>
    {title}
  </Typography>
)

export default PageTitle
