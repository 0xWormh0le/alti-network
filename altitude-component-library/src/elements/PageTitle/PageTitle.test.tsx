import React from 'react'
import { render } from '@testing-library/react'

import PageTitle from './PageTitle'

describe('PageTitle', () => {
  it('renders correctly', () => {
    const props = {
      title: 'Sample Page Title',
    }
    const { container } = render(<PageTitle {...props} />)
    expect(container).toMatchSnapshot()
  })
})
