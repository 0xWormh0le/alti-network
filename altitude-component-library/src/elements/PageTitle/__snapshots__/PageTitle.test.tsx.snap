// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle renders correctly 1`] = `
<div>
  <h1
    class="Typography Typography--h2 Typography--normal PageTitle"
  >
    Sample Page Title
  </h1>
</div>
`;
